code/modules/scripting/Implementations/Telecomms.dm

```diff
@@ -237,7 +237,7 @@ datum/signal
 				S.memory[address] = value
 
 
-	proc/tcombroadcast(var/message, var/freq, var/source, var/job, var/language)
+	proc/tcombroadcast(var/message, var/freq, var/source, var/job, var/verb, var/language)

```

Comment:
```suggestion
/datum/signal/proc/tcombroadcast(var/message, var/freq, var/source, var/job, var/verb, var/language)
```
Un-indent, too.

Reply:
Maybe absolute-path the rest of this file while you're at it, if it's not too much trouble? Would help with maintainability.

Reply:
Actually, if you absolute-path this, you'll need to absolute-path the rest to make it work, so if you don't want to do that I *guess* you could keep this relatively-pathed. There's no reason to, really, though, just more trouble in the long run.

Reply:
This part isn't my work, blame who ever wrote Telecomms.